jetty-websocket/websocket-jetty-server/src/main/java/module-info.java

```diff
@@ -28,15 +28,9 @@
     exports org.eclipse.jetty.websocket.server.config;
 
     requires jetty.servlet.api;
-    requires org.eclipse.jetty.http;
-    requires org.eclipse.jetty.server;
-    requires org.eclipse.jetty.servlet;
-    requires org.eclipse.jetty.util;
-    requires org.eclipse.jetty.webapp;
-    requires org.eclipse.jetty.websocket.core;
-    requires org.eclipse.jetty.websocket.jetty.api;
-    requires org.eclipse.jetty.websocket.jetty.common;
-    requires org.eclipse.jetty.websocket.servlet;
+    requires transitive org.eclipse.jetty.webapp;
+    requires transitive org.eclipse.jetty.websocket.jetty.common;
+    requires transitive org.eclipse.jetty.websocket.servlet;
```

Comment:
Do we need the `transitive` for websocket servlet? this is an internal package like core.

Reply:
I think so - again the reason is that there are `-servlet` classes in `-server` signatures.
Double check which one by removing `transitive`.